### DISCUSSION TITLE
Ormar to Json

### DISCUSSION THREAD

**Discussion by avico78** [collerek/ormar#207] [Q&A]
Trying to find best way convert a ormal model to Json (need to save it as file),

i.e:
```python
    customer_final = sdv_models.Customer(**customer_to_synt)  
    test_customer = customer_final.dict(exclude_primary_keys=True) 
```
The only way i manage to get a json out of this is using
```python
print(jsonable_encoder(test_customer))
```
following:
https://collerek.github.io/ormar/models/methods/#json

Im aware i could first save it to DB and then get and apply json() 
but is there any other way?

**Comment by collerek** [comment]
You don't have to save it to db, `json()` works the same way as `dict()` and has same parameters (well it even calls `dict()` under the hood). 
But of course if you need fields populated by database then you need to save it before (but don't see it in your example).
Also if you return a dict in `fastapi` it will joson-ize it for you.
